internal/pkg/constants/constants.go

```diff
@@ -34,6 +34,9 @@ const (
 	PolicyServerVerificationConfigEntry         = "verification-config"
 	PolicyServerVerificationConfigContainerPath = "/verification"
 
+	// PolicyServer PodDisruptionBudget
+	PolicyServerPodDisruptionBudgetSuffix = "pdb"
```

Comment:
Probably we can get rid of that if we name the PDB object using the Policy Server's name